app/javascript/pages/howToApply/how-to-apply.tsx

```diff
@@ -228,22 +276,44 @@ const WhatHappensNextSection = () => {
 }
 
 const HowToApply = (_props: HowToApplyProps) => {
+  const [listing, setListing] = useState<RailsSaleListing>(null)
+
+  const { router } = useContext(NavigationContext)
+
+  // identical data loading strategy to the Listing Details page
+  useEffect(() => {
+    const path = getPathWithoutLanguagePrefix(router.pathname)
+    void getListing(path.split("/")[2]).then((listing: RailsSaleListing) => {
+      if (!listing) {
+        router.push("/")
+      }
+      setListing(listing)
+    })
+  }, [router, router.pathname])
+
   return (
-    <Layout title={t("pageTitle.howToApply")}>
-      <section className="flex md:px-5">
-        <article className="markdown max-w-5xl m-auto">
-          <div className="pt-4 md:py-0 max-w-3xl">
-            <div className="my-6 md:my-12 px-5">
-              <HowLongItTakesSection />
-              <BeforeYouStartSection />
-              <HowToApplySection />
-              <WhatHappensNextSection />
+    <LoadingOverlay isLoading={!listing}>
+      <Layout title={t("pageTitle.howToApply")}>
+        <section className="flex md:px-5">
+          <article className="markdown max-w-5xl m-auto">
+            <div className="pt-4 md:py-0 max-w-3xl">
+              <div className="my-6 md:my-12 px-5">
+                {listing && (
```

Comment:
suggestion: instead of checking for the listing here, change the two instances of `listing.Application_Start_Date_Time` to `listing?.Application_Start_Date_Time` - simplifies the code

Reply:
Some counterpoints:
1. additional changes would also be needed to handle falsey params for `localizedFormat` and `formatTimeOfDay`
2. future components that use `listing` may also need to handle falsey values, I think it's more efficient to block rendering of all components until `listing` is loaded, like how it's done on the Listing Details page.

Let me know if those make sense.

Reply:
Update: I like the suggestion, if only because the markup looks a little cleaner. Authors of future components should feel free to revert if they need to.

Reply:
Yeah I was going for the cleaner markup, but it is what it is